variables.tf

```diff
@@ -125,6 +125,18 @@ variable "views" {
   }))
 }
 
+variable "materialized_views" {
+  description = "A list of objects which includes view_id, and view query"
+  default     = []
+  type = list(object({
```

Comment:
Since AFAIK materialized views *do* support range partitioning and other table features, we should probably include these:

- clustering
- time_partitioning
- range_partitioning
- expiration_time